fastlane/lanes/release.rb

```diff
@@ -91,11 +146,49 @@
   #
   desc 'Trigger a new beta build on CI'
   lane :new_beta_release do |options|
-    ios_betabuild_prechecks(options)
+    # Verify that there's nothing in progress in the working copy
+    ensure_git_status_clean
+
+    # Check out the up-to-date default branch, the designated starting point for the new beta
+    Fastlane::Helper::GitHelper.checkout_and_pull(DEFAULT_BRANCH)
```

Comment:
Do we need to check the default branch for a beta? 

If not, what do you think of removing the instruction and of moving release branch check block here, like we do in the `complete_code_freeze`?

```ruby
    # Verify that the current branch is a release branch. Notice that `ensure_git_branch` expects a RegEx parameter
    ensure_git_branch(branch: '^release/')
```

Reply:
Yep, I don't know why we would need to check out `trunk` first before making a new beta. I went ahead and removed that check and added in the `release` branch check. I also removed the `base_version` option after discussing it with Oguz on the WPAndroid PR: https://github.com/wordpress-mobile/WordPress-Android/pull/19288#discussion_r1346199025